apps/api/schedule.py

```diff
@@ -50,13 +56,26 @@ def patch(self, proposal_id):
         if set(payload.keys()) - ALLOWED_ATTRIBUTES:
             abort(400)
 
+        if "video_recording_lost" in payload and len(payload) > 1:
+            # Flagging a recording lost is mutually exclusive with other edits
+            abort(400)
+
+        was_published = proposal.is_video_published
         for attribute in ALLOWED_ATTRIBUTES:
             if attribute in payload:
                 setattr(proposal, attribute, payload[attribute] or "")
+        is_published = proposal.is_video_published
 
         db.session.add(proposal)
         db.session.commit()
 
+        if was_published is False and is_published is True:
+            send_email_for_proposal(
+                proposal,
+                reason="video-recording-published",
+                from_address=from_email("SPEAKERS_EMAIL"),
```

Comment:
Drawing attention to this for review: is this sensible, or should this use the `CONTENT_EMAIL` address (or other) instead?

Reply:
I think they both go to the same mailbox anyway so I'm not too fussed about this.